src/SAML2/XML/saml/EncryptedID.php

```diff
@@ -57,8 +42,6 @@ public function decrypt(EncryptionAlgorithmInterface $decryptor): ElementInterfa
         switch ($id) {
             case NameID::NS . ':NameID':
                 return NameID::fromXML($xml);
-            case Issuer::NS . ':Issuer':
```

Comment:
Issuer used to extend from NameID in the past, which was a mistake. Issuer is not supposed to be encryptable.